### DISCUSSION TITLE
Add a new flag/template to output a message after init

### DISCUSSION THREAD

**Discussion by sysadmiral** [twpayne/chezmoi#4079] [Ideas]
I was thinking after running `chezmoi init` for the first time there are probably some other commands that will be useful to run on a machine i.e. install some package or run some configuration.

This assumes on a new machine you do things in this sort of order:
- preliminary bootstrapping (install chezmoi)
- run `chezmoi init`
- run other commands

I think it would be handy if some special flag/templatedir exists that will output a message after init to remind you of the additional steps you might want.

I think a template would be nicer because then you can output os/machine specific messages.

Example template in `.chezmoiinitmessagetemplates` (made up dir name):
```
Post-chezmoi steps:
{{- if eq .chezmoi.os "Debian" }}
- sudo apt install pipx
- pipx install ansible-core
{{- end }}
```

Example output:
```
Post-chezmoi steps
- sudo apt install pipx
- pipx install ansible-core
```

Does this sound useful?

**Comment by twpayne** [comment]
> Does this sound useful?

Yes, but you can also achieve this with a `run_once_after_.tmpl` script, e.g.

```bash
#!/bin/bash
# run_once_after_post-chezmoi.sh.tmpl

echo Post-chezmoi steps:
{{- if eq .chezmoi.os "Debian" }}
echo '- sudo apt install pipx'
echo '- pipx install ansible-core'
{{- end }}
```

**Reply by sysadmiral** [reply]
Would that run after every invocation of Chezmoi?

**Reply by twpayne** [reply]
No, because it's `once_` script. See the [reference guide on scripts](https://www.chezmoi.io/reference/target-types/#scripts).

**Reply by sysadmiral** [reply]
perfect. sounds like I can achieve what I'm after with that. Thank you! ❤️
